iroh-ctl/src/fixture.rs

```diff
@@ -0,0 +1,44 @@
+use std::collections::HashMap;
+use std::env;
+
+use iroh::{MockApi, MockP2p};
+use libp2p::PeerId;
+
+type GetFixture = fn() -> MockApi;
+type FixtureRegistry = HashMap<String, GetFixture>;
+
+fn fixture_peer_ids() -> MockApi {
+    let mut api = MockApi::default();
+    api.expect_p2p().returning(|| {
+        let mut mock_p2p = MockP2p::default();
+
+        mock_p2p.expect_peer_ids().returning(|| {
+            let peer_id0 = PeerId::from_bytes(&[
+                0, 32, 15, 231, 162, 148, 52, 155, 40, 187, 217, 170, 125, 185, 68, 142, 156, 196,
+                145, 178, 64, 74, 19, 27, 9, 171, 111, 35, 88, 236, 103, 150, 96, 66,
```

Comment:
Please use the `FromStr` for peer ids, that makes these much more readable

Reply:
Yeah, I struggled with this before not realizing there was a trait. I've went and used `parse` now.